hadoop-hdds/docs/content/feature/Observability.zh.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "可观察性"
+weight: 8
+menu:
+main:
+parent: 特性
+summary: Ozone 的不同工具来提高可观察性
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 提供了多种工具来获取有关集群当前状态的更多信息。
+
+## Prometheus
+Ozone 原生支持 Prometheus 集成。所有内部指标（由 Hadoop 指标框架收集）都发布在 `/prom` HTTP 端点下。（例如，在 SCM 的 http://localhost:9876/prom 下）。
+
+Prometheus 端点默认打开，但可以通过`hdds.prometheus.endpoint.enabled`配置变量关闭。
+
+在安全环境中，该页面由 Prometheus 不支持的 SPNEGO 身份验证保护。要在安全环境中启用监控，可以配置特定的身份验证令牌
+
+`ozone-site.xml` 范例：
+```XML
+<property>
+   <name>hdds.prometheus.endpoint.token</name>
+   <value>putyourtokenhere</value>
+</property>
+```
+
+prometheus 设定范例：
+```YAML
+scrape_configs:
+  - job_name: ozone
+    bearer_token: <putyourtokenhere>
+    metrics_path: /prom
+    static_configs:
+     - targets:
+         - "127.0.0.1:9876" 
+```
+
+## 分布式追踪
+分布式追踪可以通过可视化端到端性能来帮助理解性能瓶颈。
+
+Ozone 使用 [jaeger](https://jaegertracing.io) 跟踪库收集跟踪，可以将跟踪数据发送到任何兼容的后端(Zipkin，…)。
+
+跟踪在默认情况下是关闭的，但是可以从 `ozone-site.xml` 打开 `hdds.tracing.enabled`
+
+```XML
+<property>
+   <name>hdds.tracing.enabled</name>
+   <value>true</value>
+</property>
+```
+
+Jager客户端可以用环境变量进行配置，如[这里](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)所记录的：
```

Comment:
```suggestion
Jaeger 客户端可以用环境变量进行配置，如[这份](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)文档所述。
```
There is a typo here in the English document for "Jager", can you help to fix it?
`Jager` -> `Jaeger`